src/Benchmarks/runtimeconfig.template.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This isn't about benchmarking. There's no earthly reason to have a long-running web project (which both BenchmarkClients and BenchmarksServer are) use regular GC. That's why we have this specified in web samples.